features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -943,6 +943,10 @@
         <PasswordProvisioningUI>/accountrecoveryendpoint/signup.do</PasswordProvisioningUI>
     </JITProvisioning>
 
+    <OutboundProvisioning>
+        <ResetProvisioningEntitiesOnConfigUpdate>{{outbound_provisioning_management.reset_provisioning_entities_on_config_update}}</ResetProvisioningEntitiesOnConfigUpdate>
```

Comment:
We don't need to add the place holders in the identity.xml. Instead, shall we can add the value?